interfaces/cython/cantera/ck2cti.py

```diff
@@ -312,14 +316,55 @@ def to_cti(self, indent=0):
         else:
             optStr = repr(options)
 
-        if self.duplicate:
-            kinstr = kinstr[:-1] + ",\n{0}options={1})".format(k_indent, optStr)
-
         if self.fwdOrders:
             order = ' '.join('{0}:{1}'.format(k,v)
                              for (k,v) in self.fwdOrders.items())
-            kinstr = kinstr[:-1] + ",\n{0}order='{1}')".format(k_indent, order)
 
+            for(k,v) in self.fwdOrders.items():
+                #Check for negative orders
+                if float(v) < 0 and NEGATIVE_ORDERS is False:
+                    #Meaning negative order detected but --allow-negative-order flag not passed to ck2cti
+                    print('\nWARNING: Reaction with negative order detected.') 
+                    print('Re-execute ck2cti with --allow-negative-orders and also execute ck2cti --help to see note on danger of negative orders.\n')
```

Comment:
Can't think of an alternative. The goal is to get some mechanisms that hitherto do not work with Cantera to work, right ?